catroid/src/main/java/org/catrobat/catroid/drone/ardrone/DroneStageActivity.java

```diff
@@ -147,7 +147,7 @@ public void onDroneBatteryChanged(int value) {
 
 		DroneControlService dcs = DroneServiceWrapper.getInstance().getDroneService();
 		if (dcs != null && (value < DroneInitializer.DRONE_BATTERY_THRESHOLD) && dcs.getDroneNavData().flying && !droneBatteryMessageShown) {
-			ToastUtil.showError(this, R.string.notification_low_battery + " " + Integer.toString(value));
+			ToastUtil.showError(this, getString(R.string.notification_low_battery) + " " + value);
```

Comment:
If you want, you can also localize this resource. Up to you :wink:

Reply:
ok